app/app.test.js

```diff
@@ -293,4 +291,27 @@ describe('GET',() => {
                 })   
            })
         });
-      })
\ No newline at end of file
+      })
+
+      describe('GET',() => {
+        describe('/api/articles', () => {
+            test('return with an array of article objects, sorted by oldest first ', async () => {
+                const response = await request(app).get('/api/articles?topic=mitch');
+                expect(200)
+                expect(response.body.articles.length).toEqual(12) // there was only 1 article that didn't have the topic of 'mitch' so 13(totalarticles) - 1 = 12
+                response.body.articles.forEach((article) => {
+                 expect(typeof article.comment_count).toBe("string")   
+                 expect(typeof article.article_id).toBe("number")   
+                 expect(typeof article.title).toBe("string")
+                 expect(article.topic).toBe("mitch")
+                })
+            })
+            test('return error when topic does not exist', async () => {
```

Comment:
Would this be a 400 though ?

If you consider usage there could be loads of topics added and removed all the time. A topic that isn't there just means it isn't there yet, rather than will never exist?

With this in mind would a 404 be more suitable?